### DISCUSSION TITLE
Battery operated Nodes showing as "Unconnected" under "Network graph"

### DISCUSSION THREAD

**Discussion by snowmangh** [zwave-js/zwave-js-ui#3103] [Q&A]
Really like to new mesh graph styles but wondering why battery operated are shown as "Unconnected". With the old graph style, I could see the interconnections. Healing does not do anything. Yet, my home automation system works perfectly. Thanks

I did search the site, and unless I am blind, could not see an answer to my question. 

![image](https://github.com/zwave-js/zwave-js-ui/assets/90811206/f7a1ad49-61a6-4f3e-8d87-7f98db3bff38)

**Comment by robertsLando** [comment]
@snowmangh we are working on a way to prevent this. The problem is that in the previous implementation we were drawing edges between nodes and their neighbors but that was totally unreliable, now we show the edges using the lwr and nlwr, the routes that the controller is actually using in order to ship/receive the packets from that node. Problem is this information is present only if the driver sends me a stat with that info, in next release I will add a method to query this info on startup. BTW Pinging a node or healing it should make it's routes visible

**Comment by snowmangh** [comment]
Understood and thanks for the explanation. All my battery powered devices are now showing in the meshed graph.
